atomicapp/providers/openshift.py

```diff
@@ -294,16 +338,24 @@ def _parse_kubeconf(self, filename):
             user:
                 token: abcdefghijklmnopqrstuvwxyz0123456789ABCDEF
         """
```

Comment:
Should move this example expected file format to the _parse_kubeconf_data function now